Composer/packages/lib/indexers/src/validations/expressionValidation/validation.ts

```diff
@@ -20,6 +20,27 @@ export const addReturnType = (currentType: number, newType: number) => {
   return currentType | newType;
 };
 
+export const trimExpression = (expression: string): string => {
+  let result = expression.trim();
+  if (result.startsWith('=')) {
+    result = result.substring(1);
+  }
+
+  result = result.trim();
+
+  if (result.startsWith('$')) {
+    result = result.substring(1);
+  }
+
+  result = result.trim();
```

Comment:
I think this could be reduced into one if, and also we can put the .trim() method inside the if

```
if (result.startsWith('=') || result.startsWith('$')) {
    result = result.substring(1).trim();
  }

if (result.startsWith('{') && result.endsWith('}')) {
    result = result.substring(1, result.length - 1);
  }

return result.trim();
```

Reply:
I did it sequentially to include cases like `=${abc()}` even though I'm not sure it's possible to receive something like that.